src/panels/config/logs/error-log-card.ts

```diff
@@ -77,17 +111,50 @@ class ErrorLogCard extends LitElement {
                     ></ha-icon-button>
                   </div>
                 </div>
-                <div class="card-content error-log">${this._logHTML}</div>
+                <div class="card-content error-log">
+                  <ha-ansi-to-html .content=${filteredLog}> </ha-ansi-to-html>
```

Comment:
This means that for every line that is added, we parse the entire log now from ansi to html. Can we instead only parse the new line added?

Reply:
Yes I think that shouln't be a problem.